src/server/rdb_load.cc

```diff
@@ -58,6 +58,7 @@ ABSL_DECLARE_FLAG(int32_t, list_compress_depth);
 ABSL_DECLARE_FLAG(uint32_t, dbnum);
 ABSL_DECLARE_FLAG(bool, list_experimental_v2);
 ABSL_FLAG(bool, rdb_load_dry_run, false, "Dry run RDB load without applying changes");
+ABSL_FLAG(bool, ignore_expiry, false, "Ignore Key Expiry when loding from RDB snapshot");
```

Comment:
lets call it `rdb_ignore_expiry` so it will be clearer that it affects rdb loading only.